canonical-web-and-design/jaas.ai
# Update the homepage hero with Suru

## Done
Updated the homepage hero to use the new suru design. Cleaned out some old/unused code. Refactored the suru sass pattern to have top and bottom modifiers.

## QA
- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- Check that the new hero section matched [the design](https://app.zeplin.io/project/5cf8ca699ee7bf1df9082012/screen/5cf8ca4478761e1b1ba07cbf)
- Check on small screens

## Details
Fixes https://github.com/canonical-web-and-design/jaas.ai/issues/323

## Screenshots
![Screenshot_2019-06-27 Jujucharms Juju](https://user-images.githubusercontent.com/1413534/60261345-8e093400-98d3-11e9-801a-5480d0a2375d.png)
